pkg/infra/tracing/tracing.go

```diff
@@ -33,7 +34,16 @@ func ProvideService(cfg *setting.Cfg) (*TracingService, error) {
 		return ts, ts.initGlobalTracer()
 	}
 
-	return ts, nil
+	ots := &OpentelemetryTracingService{
+		Cfg: cfg,
+		log: log.New("tracing"),
+	}
+
+	if err := ots.parseSettingsOpentelemetry(); err != nil {
+		return nil, ots.initOpentelemetryTracer()
```

Comment:
Something interesting is going on here. So the OTel settings can't be parsed - but we still create an OTel tracer? ;)